File: Search/C++/linear.cpp
#include<iostream>
using namespace std;
bool search(int a[],int x,int n)
{
for(int i=0;i<n;i++)
if(a[i]==x){
return 1;
}
return 0;
}
int main()
{
int a[]={1,2,3,4,5,6};
if(search(a,4,6))cout<<"4 is found in array/n"<<endl;
else cout<<"4 not found"<<endl;
}

File: Search/linear.exs
defmodule LinearSearch do
  def search([], _target), do: false
  def search([head  | _target], target) when head == target, do: true
  def search([_head | tail], target), do: search(tail, target)
end

items = [5, 10, 34, 18, 6, 7, 45, 67]
target = 10
if LinearSearch.search(items, target) do
  IO.puts "Found"
else
  IO.puts "Not found"
end


